tailwind.config.js

```diff
@@ -17,14 +17,27 @@ export default {
                 'accent-blue': '#253F9C',
                 'fade-orange':'#E5C090',
                 'fade-white': "#F7F7F7",
-                'fade-blue-white':'#F4F8FF'
+                'fade-blue-white':'#F4F8FF',
+                'light-gray': '#ddd',
+                'light-brown':'#696969'
             },
             borderRadius: {
                 '4xl': '1.75rem',
                 '5xl': '2rem'
             },
             fontSize: {
                 xxs: '0.625rem'
+            },
+            spacing :{
+                '26.75':'6.668rem',
+                '29':'-7.25rem',
```

Comment:
Why do you need this? why negative spacing? please don't add inside spacing add it where you needed like margin, padding etc